api/.env.example

```diff
@@ -5,4 +5,5 @@ COGNITO_REDIRECT_URI=http://localhost:4040/signin
 COGNITO_USER_POOL_ID=
 SECRET_KEY=
 COGNITO_ACCESS_ID=
-COGNITO_ACCESS_KEY=
\ No newline at end of file
+COGNITO_ACCESS_KEY=
+URL=http://localhost:4040
```

Comment:
A more descriptive name would help me understand the purpose of this variable.

Reply:
+1